ReleaseNotes.md

```diff
@@ -1,3 +1,3 @@
 # Bluedot React Native Plugin release notes
 
-- Fixed null CustomData in enterZone for iOS
+- Integrated with Bluedot iOS SDk v15.6.8
```

Comment:
Seems really odd to me that we're not keeping the release note history, but seems to have been standard in this repo.

Reply:
Good point, we can start keeping it from next release.